### DISCUSSION TITLE
How to configure Philips Hue Motion Sensor sensitivity

### DISCUSSION THREAD

**Discussion by nexro567** [Koenkk/zigbee2mqtt#8015] [Devices]
Hi Github users,

I have a Philips Hue Motion sensor whose motion sensitivity is very high.
On the Zigbee2MQTT website I can find that I can adjust the sensitivity via a code. I just can't figure it out and need help.

Can someone help me? I want to change the sensitivity value from "high" to "low".
I've tried adjusting Zigbee2MQTT's configuration or Home Assistant's configuration, but both don't output the `/get` in the log files.

https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/9290019758.md

Example:
To write (/set) a value publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"motion_sensitivity": NEW_VALUE}`. The possible values are: low, medium, high.

**Comment by Koenkk** [comment]
This can be done via the frontend: 

![image](https://user-images.githubusercontent.com/2892853/125312608-b9cdaa00-e334-11eb-85f8-033a8ebeedab.png)

**Reply by nexro567** [reply]
Thanks @Koenkk. Never knew that there was a fronted.
